Sources/SectionScrubber.swift

```diff
@@ -224,7 +224,11 @@ public class SectionScrubber: UIView {
             return indexPath
         }
         for indexPath in collectionView.indexPathsForVisibleSupplementaryElementsOfKind(UICollectionElementKindSectionHeader) {
-            let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)
+            #if swift(>=2.3)
+                let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)!
```

Comment:
We could make the view an optional above the #if clouse. Fill it with optional result in >=swift 2.3, fill it with view in swift lower than that. The function returns an optional indexpath anyways :v: